src/buildercore/terraform.py

```diff
@@ -816,6 +818,46 @@ def _render_eks_workers(context, template):
         }
     })
 
+    if context['eks']['helm']:
```

Comment:
This function is massive (required many clicks to get to the function signature). Maybe it would deserve being broken up?

Reply:
This part should definitely be extracted